projects/application/HashMix-Lending-en.md

```diff
@@ -0,0 +1,62 @@
+## Project Name
+
+HashMix 
+
+## Category 
+
+Application
+
+## Project Description
+
+HashMix proposes a tokenization scheme suitable for arbitrary hash power, implements a set of universal decentralized cloud hash power trading, lending, and swapping protocols to solve hash power fraud, hash power oversold, over-simplified product design, and lack of liquidity and flexibility in the current market, enables the free redistribution of global hash power.
+
+The HashMix project got support from some of the top investment institutes, including HashKey Capital, Fenbushi Capital, LongHash Ventures, GBV, Kenetic Capital, and so on.
+
+The first product HashMix launched is HashMix Lending, a lending tool helping Filecoin miners and FIL holders. Miners can pledge their miner account to borrow FIL to seal more sectors, while FIL holders can deposit FIL to receive an awesome interest. 
+
+## Project Status
+
+HashMix Lending has launched weeks ago, currently 286k FIL have been processed by HashMix platform.
+
+## Previews
+Pic
+
+## Target Audience
+Filecoin miners who need FIL to seal more storage power and FIL holders who want to fully utilize their assets.
+
+## Rough estimated user base (if applicable)
+Every miner and holder could benefit from HashMix platform.
+
+## Github repo
+https://github.com/HashMixProject
+
+
+## Website
+https://hashmix.org/
+https://app.hashmix.org/
+
+## Docs
+Coming soon.
+
+
+## Team Info
+The HashMix team is anonymous.
+
+
+### Team Size  
+20
+
+## How the community can engage
```

Comment:
Please fix the formating issue ?

Reply:
The file has been updated. Thanks for pointing out.